pkg/storaged/dialog.jsx

```diff
@@ -1022,9 +1022,9 @@ export const SizeSlider = (tag, title, options) => {
 
 export const BlockingMessage = (usage) => {
     const usage_desc = {
-        pvol:                  _("physical volume of LVM2 volume group"),
-        mdraid:                _("member of RAID device"),
-        vdo:                   _("backing device for VDO device"),
+        pvol: _("physical volume of LVM2 volume group"),
+        mdraid: _("member of RAID device"),
+        vdo: _("backing device for VDO device"),
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17876-20221109-102136-9fe18085-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)